server/helpers/identification.js

```diff
@@ -0,0 +1,14 @@
+import { User } from '../database/models';
+import responseMessage from './responseMessage';
+
+const getUserId = async (param, response) => {
```

Comment:
rename this function, and ensure to name the file after its default export

Reply:
Done